src/main/java/it/eng/idsa/businesslogic/processor/sender/registration/AbstractCreateRegistrationMessage.java

```diff
@@ -39,16 +41,24 @@ public void process(Exchange exchange) throws Exception {
 		Message connectorAvailable = getConnectorMessage();
```

Comment:
We need to be careful here, with other PR (broker autosync) since in that one we did renaming that corresponds better with what variables do in code